scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -263,7 +300,21 @@
                            :instance.status/failed} instance-status)
                     [[:db/add instance :instance/end-time (now)]])
                   (when progress
-                    [[:db/add instance :instance/progress progress]])]))))
+                    [[:db/add instance :instance/progress progress]])
```

Comment:
Do we update Datomic every time we receive a status update with progress?  How often can a task update the status?  How many status updates per unit time can we handle with the current architecture?